azure-devops/projects/pagopa-projects/pagopa-payments-tests.tf

```diff
@@ -0,0 +1,56 @@
+variable "pagopa-payments-tests" {
+  default = {
+    repository = {
+      organization    = "pagopa"
+      name            = "pagopa-payments-tests"
+      branch_name     = "main"
+      pipelines_path  = ".devops"
+      yml_prefix_name = null
+    }
+    pipeline = {
+      enable_api_tests = true
+    }
+  }
+}
+
+locals {
+  # global vars
+  pagopa-payments-tests-variables = {
+    cache_version_id = "v1"
+    default_branch   = var.pagopa-payments-tests.repository.branch_name
+  }
+  # global secrets
+  pagopa-payments-tests-variables_secret = {
+
+  }
+  # api-tests vars
+  pagopa-payments-tests-variables_api_tests = {
+    danger_github_api_token = "skip"
+  }
+  # api-tests secrets
+  pagopa-payments-tests-variables_secret_api_tests = {
+
+  }
+  
+module "pagopa-payments-tests_api_tests" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_code_review?ref=v1.0.0"
```

Comment:
@infantesimone is this a code review pipeline?

Reply:
it is similar, the goal is to run a docker container for api test, but the name of yaml file is different `code-review-pipelines.yml`, so I can't use this module, right? Otherwise, I can rename the yaml file

Reply:
For now we can consider it a `code-review` pipeline, I changed the name. Thanks

Reply:
Thanks Simone, yes yaml name it's hard coded.
Anyway u can run the pipeline manually

Reply:
other problems? 😀

Reply:
yes I tried to apply and pagopa-git-hub bot does not have required permission on github repo

Reply:
pagopa-github-bot requires admin role

Reply:
pagopa-github-bot added as admin, thanks Pasquale